chat/base.py

```diff
@@ -135,24 +141,32 @@ def main(
     """
     precision = precision or get_default_supported_precision(training=False)
 
+    plugins = None
+    if quantize is not None and quantize.startswith("bnb."):
+        if "mixed" in precision:
+            raise ValueError("Quantization and mixed precision is not supported.")
```

Comment:
I think we should eventually check this in Fabric's connector directly (no blocker ofc).

Reply:
https://github.com/Lightning-AI/lightning/pull/18671 would raise an error already